### PR TITLE
21273 - Added alert for businesses in the process of being dissolved

### PR DESCRIPTION
**Pull request by JazzarKarim**
*Issue #:* /bcgov/entity#21273

*Description of changes:*
- Added an alert for businesses that are in the process of being dissolved
- Misc. Fixes
- Unit test fixes

business in the process of dissolution

Designs followed: https://preview.uxpin.com/4648cf2627971eb4281b7ccc7c1258224b3d8084#/pages/164129752/simulate/sitemap

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: search-ui/src/resources/contact-info/registries-info.ts [hunk 1]
```diff
@@ -9,9 +9,9 @@ export const RegistriesInfo: ContactInfoI[] = [
   },
   {
     icon: 'mdi-phone',
-    href: 'tel:+1-250-952-0568',
+    href: 'tel:+1-250-387-7848',
```
Comment (JazzarKarim) [inline search-ui/src/resources/contact-info/registries-info.ts:12]: I noticed that this is super out of date. If you look at the designs (https://preview.uxpin.com/4648cf2627971eb4281b7ccc7c1258224b3d8084#/pages/164129752/simulate/sitemap) and what we have in other repos, this is the number being used for "Victoria Office".

Comment (kialj876) [reply search-ui/src/resources/contact-info/registries-info.ts:12]: Good to know -- I will double check the other UIs BTR is managing

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://registry-search-dev--pr-261-a4qv92k2.web.app

**Comment by JazzarKarim** [comment]
You can search for "BC0878513" for a business that's in the process of being dissolved.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://registry-search-dev--pr-261-a4qv92k2.web.app

**Comment by kialj876** [comment]
Nice work 👍

**Review by kialj876 — APPROVED** [review]

**Pull request merged**